zebra-chain/src/sprout/tree.rs

```diff
@@ -164,8 +176,13 @@ impl From<Vec<NoteCommitment>> for NoteCommitmentTree {
 impl NoteCommitmentTree {
     /// Get the Jubjub-based Pedersen hash of root node of this merkle tree of
     /// commitment notes.
-    pub fn hash(&self) -> [u8; 32] {
-        self.root.0
+    pub fn root(&self) -> Root {
+        self.root
+    }
+
+    /// Add a note commitment to the tree.
+    pub fn append(&mut self, _cm: &NoteCommitment) {
+        // TODO: https://github.com/ZcashFoundation/zebra/issues/2485
```

Comment:
Let's split the sprout code into another branch, rather than ignoring new sprout note commitments, and producing invalid anchors?

Reply:
Also let's make sure we fail the tests if we accidentally call unimplemented code.
```suggestion
        todo!("implement sprout note commitment trees #2485");
```

Reply:
Removed in  d3a099f9b. Created branch `tree-state-sprout`, mentioned it in #2485 

Added todo! in 3d7d252